components/movieNight.py

```diff
@@ -0,0 +1,173 @@
+suggestion_database = Suggestions(SUGGESTION_DB_PATH)
+movie_night_group = discord.app_commands.Group(name="movie", description="Movie night slash commands")
+
+
+# Code that needs to execute every time the bot starts
+def init():
+    upcomingMovie.update_event_description(False)
+    upcomingMovie.start_pick_reminder()
+    eventReminder.start_event_reminder()
+
+
+# add-suggestion command
+# Creates a modal that takes movie name, genre and reason for picking as input
+@movie_night_group.command(name="add-suggestion", description="Suggest a movie using a popup")
+async def add_suggestion(interaction: discord.Interaction):
+    await interaction.response.send_modal(SuggestionModal())
+
+
+# add-suggestion modal
+class SuggestionModal(discord.ui.Modal, title="Suggest a Movie"):
+    movie_name = discord.ui.TextInput(label="Movie Name")
+    movie_genre = discord.ui.TextInput(label="Movie Genre")
+    movie_reason = discord.ui.TextInput(label="Reason for Picking", style=discord.TextStyle.paragraph)
+
+    async def on_submit(self, interaction: discord.interactions):
+        suggested_movie = Movie(self.movie_name.value, self.movie_genre.value, self.movie_reason.value)
+        sender = interaction.user.name
+        reply = suggestion_database.add_suggestion(sender, suggested_movie)
+
+        await interaction.response.send_message(embed=reply, delete_after=ut.DELETE_AFTER_SECONDS)
+
+
+# remove-suggestion command
+# Removes movie from user's suggestion list
+@movie_night_group.command(name="remove-suggestion", description="Remove a previously suggested movie")
+@discord.app_commands.describe(movie_name="Name of the movie to be removed from suggestion list")
+async def remove_suggestion(interaction: discord.Integration, movie_name: str):
+    reply = suggestion_database.remove_suggestion(interaction.user.name, movie_name)
+    await interaction.response.send_message(embed=reply, delete_after=ut.DELETE_AFTER_SECONDS)
+
+
+# list-suggestions command
+# Lists all suggestions of a specific user
+@movie_night_group.command(name="list-suggestions", description="List everyone's suggestions")
+async def list_suggestions(interaction: discord.Integration):
+    reply = suggestion_database.get_list_embed()
+    await interaction.response.send_message(embed=reply, delete_after=ut.DELETE_AFTER_SECONDS)
+
+
+# view-suggestion command
+# View a specific suggestion of a specific user
+@movie_night_group.command(name="view-suggestion", description="View a specific suggestion")
+@discord.app_commands.describe(user="User's movie-suggestion list")
+@discord.app_commands.describe(movie_name="View more details on a specific movie")
+async def view_suggestion(interaction: discord.Integration, user: discord.Member, movie_name: str):
+    if not user:
+        user = interaction.user
+    reply = suggestion_database.get_suggestion_embed(user.name, movie_name)
+    await interaction.response.send_message(embed=reply, delete_after=ut.DELETE_AFTER_SECONDS)
```

Comment:
Let's not delete this message for the same reason as above.

Reply:
Same reply as above.